### PR TITLE
Check system and SSSE3 (with three S, not two)

### PR DESCRIPTION
**Pull request by Megamouse**
add system_info.h

add system info to log
add SSSE3 Error message to main_window

### CODE REVIEW

File: rpcs3/rpcs3qt/system_info.h [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+		std::string s_sysInfo = fmt::format("%s | SSSE3 %s", std::regex_replace(brand, std::regex("^ +"), ""), supports_ssse3 ? "Supported" : "Not Supported");
+#ifdef _WIN32
+		SYSTEM_INFO sysInfo;
+		GetNativeSystemInfo(&sysInfo);
+		MEMORYSTATUSEX memInfo;
+		memInfo.dwLength = sizeof(memInfo);
+		GlobalMemoryStatusEx(&memInfo);
+		s_sysInfo += fmt::format(" | %d Threads | %.2f GB RAM", sysInfo.dwNumberOfProcessors, (float)memInfo.ullTotalPhys / std::pow(1024.0f, 3));
+#endif
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/system_info.h]: Someone add Linux code here

**Comment by MonokelPinguin** [comment]
Something like this should work: [https://github.com/MonokelPinguin/rpcs3/commit/966bc1fa8f620f56db94ae33911658780794b131](https://github.com/MonokelPinguin/rpcs3/commit/966bc1fa8f620f56db94ae33911658780794b131)

**Comment by Megamouse** [comment]
thx @MonokelPinguin 

Let's see if it works

**Comment by Megamouse** [comment]
@hcorion try memory again

**Comment by hcorion** [comment]
Yep, works good, LGTM

**Pull request merged**